lib/linker.version

```diff
@@ -83,6 +83,8 @@ MOSQ_1.5 {
 	global:
 		mosquitto_subscribe_simple;
 		mosquitto_subscribe_callback;
+		mosquitto_publish_single;
+		mosquitto_publish_multiple;
```

Comment:
Could you please add these to the MOSQ_1.7 section at the bottom? That indicates they were added in version 1.7.